edd-blockonomics.php

```diff
@@ -493,13 +493,16 @@ public function listener()
           $existing_status = $order['status'];
           $timestamp = $order['timestamp'];
           $time_period = edd_get_option("edd_blockonomics_timeperiod", 10) *60;
-
+          $network_confirmations = edd_get_option("edd_blockonomics_confirmations", 2);
+          if($network_confirmations == 'zero'){
+            $network_confirmations = 0;
+          }
           if ($status == 0 && time() > $timestamp + $time_period)
           {
             $minutes = (time() - $timestamp)/60;
             edd_record_gateway_error(__("Warning: Payment arrived after $minutes minutes. Received BTC may not match current bitcoin price", 'edd-blockonomics'));
           }
-          elseif ($status == 2)
+          elseif ($status >= $network_confirmations)
```

Comment:
We need to avoid duplicate execution of this code. It may execute once on 0 status and then again on status 1 and 2. This is cause multiple payment notes which are uncessary

Reply:
Fixed with commit 4b1d54d

Reply:
Updated 7123c43